vllm/outputs.py

```diff
@@ -134,27 +134,43 @@ def __init__(
         self.encoder_prompt_token_ids = encoder_prompt_token_ids
         self.num_cached_tokens = num_cached_tokens
 
-    def add(self, next_output: "RequestOutput") -> None:
+    def add(self,
+            next_output: "RequestOutput",
+            aggregate: bool = True) -> None:
```

Comment:
Maybe remove the default value here, better for it to always be explicit in this case.